skimage/morphology/tests/test_misc.py

```diff
@@ -395,6 +400,7 @@ def test_priority(self):
         desired = np.array([0, 1, 0, 1])
         assert_array_equal(result, desired)
 
+    @pytest.mark.thread_unsafe
     @pytest.mark.parametrize("order", ["C", "F"])
     def test_out(self, order):
```

Comment:
Why isn't this test thread-safe?

Reply:
I think it was related to the aforementioned Cython issues.